ui/app/AppLayouts/Wallet/popups/AddEditSavedAddressPopup.qml

```diff
@@ -273,7 +276,16 @@ StatusModal {
 
             d.resolvingEnsNameInProgress = false
             d.address = resolvedAddress
-            d.checkForAddressInputOwningErrorsWarnings()
+            try { // allows to avoid issues in storybook without much refactoring
+                d.checkForAddressInputOwningErrorsWarnings()
+            }
+            catch (e) {
+            }
+
+            if (!d.addressInputHasError)
+                networkSelector.state = "networksHidden"
```

Comment:
@IvanBelyakoff could you set this as readonly property?

`readonly property string networksHiddenState: "networksHidden"`